pkg/chartutil/load.go

```diff
@@ -135,6 +135,9 @@ func LoadFiles(files []*BufferedFile) (*chart.Chart, error) {
 			return c, errors.New("values.toml is illegal as of 2.0.0-alpha.2")
 		} else if f.Name == "values.yaml" {
 			c.Values = &chart.Config{Raw: string(f.Data)}
```

Comment:
Do you also need to set `c.Values.Type` here?

Reply:
The type will default to `chart.ConfType_VALUES` since its initialized to `0` but I can make it explicit if that would be clearer?